pkg/liquidity-mining/contracts/BalancerMinter.sol

```diff
@@ -35,9 +36,6 @@ contract BalancerMinter is IBalancerMinter, ReentrancyGuard, EIP712 {
     // minter -> user -> can mint?
     mapping(address => mapping(address => bool)) private _allowedMinter;
 
-    // Signature replay attack prevention for each user.
-    mapping(address => uint256) internal _nextNonce;
-
     // solhint-disable-next-line var-name-mixedcase
     bytes32 private immutable _SET_MINTER_APPROVAL_TYPEHASH = keccak256(
```

Comment:
```suggestion
    bytes32 private constant _SET_MINTER_APPROVAL_TYPEHASH = keccak256(
```
While we're touching these files, we can switch these to constant as immutable doesn't save us any gas here (which was the original justification afaik).